provisioning/miniprov/hare_mp/main.py

```diff
@@ -635,7 +663,13 @@ def get_config_dir(url) -> str:
 
 def cleanup(args):
     try:
+        url = args.config[0]
+        conf = ConfStoreProvider(url)
+        utils = Utils(conf)
+        cns_utils = ConsulUtil()
         kv_cleanup()
+        cleanup_node_facts(utils, cns_utils)
+        cleanup_node_disks(utils, cns_utils)
```

Comment:
`cleanup_node_facts(utils, cns_utils)
cleanup_node_disks(utils, cns_utils)`
these two functions are also doing kv cleanup. Would be good if both  the calls are moved in `kv_cleanup()`

Reply:
Yes, moved these two function calls into `kv_cleanup()`